macros/ano_macros.cfg

```diff
@@ -1128,6 +1132,15 @@
     [/if]
 #enddef
 
+# Use this for Bor Cryne and Uri van Roe:
+#define ANO_ENEMY_LEADER_TRAITS
+    [modifications]
+        {TRAIT_STRONG}
+        {TRAIT_RESILIENT}
+        {TRAIT_HEALTHY}
+    [/modifications]
+#enddef
+
```

Comment:
yeah I was thinking of this more from a gameplay perspective than a flavor perspective; the point is to make them harder to kill, to make it less likely that the player triggers the error handler in Barnon, and to make the final scenario more of a boss fight against them

Reply:
Oh yeah for sure I just wonder if it would be too much work to just give them a special trait that does the same thing but has a different description. Like "bullheaded" or something. Doesn't really matter either way.

Reply:
I don't think I've actually made any of my own custom traits yet; I'll have to go take a look at how to do that...

Reply:
Okay you can just leave it if you like.

Reply:
well for now at least; might be worth coming back to later in a future release... (resolving for the purposes of this PR, though)

Reply:
(Issue #44 is to track this suggestion for future purposes)